Yoshi/Yoshi/Utility/AppBundleUtility.swift

```diff
@@ -8,27 +8,51 @@
 
 import UIKit
 
-let appDisplayNameDictionaryKey = "CFBundleName"
-let appVersionNumberDictionaryKey = "CFBundleShortVersionString"
-let appBuildNumberDictionaryKey = "CFBundleVersion"
-
+/// Utility for retrieving items from the App Bundle
 internal class AppBundleUtility: NSObject {
 
+    /**
+     The application version + build number text.
+
+     - returns: The application version.
+     */
     class func appVersionText() -> String {
-        return "\(AppBundleUtility.appDisplayName())"
-            + " | Version \(AppBundleUtility.appVersionNumber())"
+        return "Version \(AppBundleUtility.appVersionNumber())"
```

Comment:
It's still there; it's just been moved to the navigation title of the debug view controller. The version is displayed below the app icon in the table view header.